pkg/minikube/logs/logs.go

```diff
@@ -186,5 +190,15 @@ func logCommands(r cruntime.Manager, bs bootstrapper.Bootstrapper, length int, f
 	}
 	cmds[r.Name()] = r.SystemLogCmd(length)
 	cmds["container status"] = cruntime.ContainerStatusCommand()
+
+	cfg, err := config.Load(viper.GetString(config.ProfileName))
```

Comment:
Nicely done! Thank you!

So, the frustrating part is that the configuration for this profile is already loaded, but the logs package does not have access to it. If it isn't too much trouble on your part, can you help us get that addressed? It'll be cleaner and faster that way.

I know I said differently in the issue, but lets start by moving this addition to the `kubeadm` bootstrapper, since it's the only one that guarantees where the kubelet is installed. Start my moving your code here:
https://github.com/kubernetes/minikube/blob/729642ef3337aaebda7ffb5b30db522cd6023bde/pkg/minikube/bootstrapper/kubeadm/kubeadm.go#L113

Now, to plumb in our missing configuration (`config.ClusterConfig`). Some packages name this variable `cfg`, some name it `cc'. I personally find `cc` less confusing, but use whatever you happen to see in the package you are modifying.  First pass the `cfg` variable to `logs.Output`:

https://github.com/kubernetes/minikube/blob/729642ef3337aaebda7ffb5b30db522cd6023bde/cmd/minikube/cmd/logs.go#L109

Then `logs.logCommands`, where we are now:

https://github.com/kubernetes/minikube/blob/729642ef3337aaebda7ffb5b30db522cd6023bde/pkg/minikube/logs/logs.go#L158

Then to `bs.LogCommands`:

https://github.com/kubernetes/minikube/blob/729642ef3337aaebda7ffb5b30db522cd6023bde/pkg/minikube/logs/logs.go#L196

If this is all too much, please let me know.

Reply:
I can definitely try. If I need help along the way, I will ask on the slack channel.